### PR TITLE
Update docker-compose.md

### PR DESCRIPTION
**Pull request by gmerritt**
* Added a more precise indication of where to modify the docker-compose-production.yml to point to a config.json
* Added note regarding configuration requirement of external postgres

### CODE REVIEW

**Comment by gmerritt** [comment]
I have reviewed and hereby sign the CLA

**Comment by gmerritt** [comment]
recheck

File: docs/running-in-production/docker-compose.md [hunk 1]
```diff
@@ -15,13 +15,13 @@ Then access PrairieLearn from port `3000`.
 PrairieLearn can be configured by a `config.json` in the root of the repository.
 
 - First make the file `config.json` in your root repository.
-- Add the following line to `docker-compose-production.yml` under `volumes`:
+- Add the following line to `docker-compose-production.yml` under `services:pl:volumes`:
```
Comment (nwalters512) [inline docs/running-in-production/docker-compose.md:18]: I think this is a more conventional way to describe nested properties in YAML documents:

```suggestion
- Add the following line to `docker-compose-production.yml` under `services.pl.volumes`:
```

File: docs/running-in-production/docker-compose.md [hunk 2]
```diff
@@ -15,13 +15,13 @@ Then access PrairieLearn from port `3000`.
...
 
 ```sh
 - ./config.json:/PrairieLearn/config.json
 ```
 
-The `config.json` file should contain appropriate overrides for the keys in [`lib/config.js`](`https://github.com/PrairieLearn/PrairieLearn/blob/master/apps/prairielearn/src/lib/config.js`). At a minimum, you'll probably want to update the various `postgres*` options to point it at your database.
+The `config.json` file should contain appropriate overrides for the keys in [`lib/config.js`](`https://github.com/PrairieLearn/PrairieLearn/blob/master/apps/prairielearn/src/lib/config.js`). At a minimum, you'll probably want to update the various `postgres*` options to point it at your database. Note that PrairieLearn is configured to connect to postgress unencrypted (no SSL), so ensure that your postgres instance permits such connections from your PrairieLearn host.
```
Comment (nwalters512) [inline docs/running-in-production/docker-compose.md:24]:
```suggestion
The `config.json` file should contain appropriate overrides for the keys in [`lib/config.js`](`https://github.com/PrairieLearn/PrairieLearn/blob/master/apps/prairielearn/src/lib/config.js`). At a minimum, you'll probably want to update the various `postgres*` options to point it at your database. Note that PrairieLearn is configured to connect to Postgres unencrypted (no SSL), so ensure that your Postgres instance permits such connections from your PrairieLearn host.
```

**Comment by nwalters512** [comment]
@gmerritt did you plan on pushing fixes for the comments you resolved? Let me know if you don't have the bandwidth, I'm happy to do that on your behalf so we can land this.

**Comment by gmerritt** [comment]
Hi Nathan,

Eep! I've been out a few days & also swamped...thank you for the
ping...yes, please, if you could follow it through, that would be amazing!!
Thank you!

-Greg

--

Greg Merritt ***@***.***>

UC Berkeley PrairieLearn Pilot <https://cbt-dev.berkeley.edu/> on AWS

UC Berkeley DataHub <https://datahub.berkeley.edu/> on GCP

**Comment by gmerritt** [comment]
D'oh -- yeah ok, I think I just missed the commit suggestion etc previously....I just did an unresolve & commit suggestion....ehhh and one is outdated...hope I haven't made a mess of this simple change.... :/

**Comment by nwalters512** [comment]
@gmerritt you're good! No merge conflicts, and it looks like my suggestions were applied successfully. I approved the CI run and will merge once it's green ✅

**Review by nwalters512 — APPROVED** [review]

**Pull request merged**